docs/api.md

```diff
@@ -335,6 +336,56 @@ Content-Length: 0
 Content-Type: text/plain; charset=utf-8
 ```
 
+
+### migrate
+
+This api used to migrate the data from one backend to another.
+
+```
+Path:    /v1/shard/migrate
+Method:  POST
+Request: {
+			"to-flavor":                 "Destination db flavor, like mysql/mariadb/radondb, default mysql"
+			"from":                      "Source MySQL backend(host:port)",                                   [required]
+			"from-user":                 "MySQL user, must have replication privilege",                       [required]
+			"from-password":             "MySQL user password",                                               [required]
+			"from-database":             "Source database",                                                   [required]
+			"from-table":                "Source table",                                                      [required]
+			"to":                        "Destination MySQL backend(host:port)",                              [required]
+			"to-user":                   "MySQL user, must have replication privilege",                       [required]
+			"to-password":               "MySQL user password",                                               [required]
+			"to-database":               "Destination database",                                              [required]
+			"to-table":                  "Destination table",                                                 [required]
+			"radonurl":                  "Radon RESTful api(default: http://peer-address)",
+			"cleanup":                   "Cleanup the from table after shifted(defaults false)",
+			"mysqldump":                 "mysqldump path",
+			"threads":                   "shift threads num(defaults 16)",
+			"behinds":                   "seconds behinds num(default 2048)",
```

Comment:
`behinds` means `position behinds` here, not time

Reply:
> `behinds` means `position behinds` here, not time

done